DMCompiler/DMStandard/Types/World.dm

```diff
@@ -12,6 +12,7 @@
 	var/timeofday
 	var/realtime
 	var/tick_lag = 1
+	var/cpu = 100
```

Comment:
Didn't seem like the best idea to leave it null on the first tick, so instead lets presume the first tick is costly.

Reply:
`world.cpu` starts off as a 0

Reply:
```suggestion
	var/cpu = 0
```